doc/changes/0.10.4.rst

```diff
@@ -2,19 +2,29 @@
 
 .. include:: names.rst
 
-0.10.4.dev
-==========
+0.10.4
+======
+
+**Released April 2024**
+
+HIGHLIGHTS
+----------
+
+- :bdg-dark:`Code` Use the appropriate HTML templates for :class:`nilearn.maskers.MultiNiftiMapsMasker` and :class:`nilearn.maskers.MultiNiftiLabelsMasker` reports that would lead to empty reports for ``MultiNiftiMapsMasker`` (:gh:`4310` by `Rémi Gau`_).
```

Comment:
I think we should just probably say that this is mainly a fix release that fixes bug introduced in the previous release or older bugs (several of them in the nilearn reports)